tests/controllers/QueryContrtollerTest.php

```diff
@@ -0,0 +1,291 @@
+<?php
```

Comment:
Typo in filename "tests/controllers/QueryContrtollerTest.php" > QueryControllerTest